lib/recurly/hosted-fields.js

```diff
@@ -29,13 +30,55 @@ export class HostedFields extends Emitter {
     this.errors = [];
     this.initQueue = [];
     this.readyState = 0;
+    this.onTabNext = this.onTabNext.bind(this)
+    this.onTabPrevious = this.onTabPrevious.bind(this)
     this.configure(options);
     this.inject();
     this.on('hostedField:state:change', this.update.bind(this));
     this.on('bus:added', bus => {
       this.bus = bus;
       this.fields.forEach(hf => bus.add(hf));
     });
+
+    this.on('hostedField:tab:previous',this.onTabPrevious)
+    this.on('hostedField:tab:next',this.onTabNext)
+  }
+
+  onTabNext(body) {
+    let type = body.type
+    let field = this.fields.find(function(f){ return f.type === type })
+    let allFields = document.querySelectorAll("[data-recurly]")
```

Comment:
It doesn't, but it would be able to tell you the html elements of the hosted fields in that instance. hostedFields.fields contains the array of HostedField instances, which each ref their containers. You could then crawl the dom tree for the previous and next tabbing targets in the form